testproject/testproject/settings/demo.py

```diff
@@ -0,0 +1,10 @@
+from .base import *  # noqa
+
+DEBUG = False
+
+INTERNAL_IPS = []
```

Comment:
@benjaoming Not sure if here we'll leave a fixed value or read it from the environment var.

Reply:
These parts are injected by the deployment script so they can be ignored. Unless you know a good default that's needed for local deployments of the demo :+1: